src/components/checkbox/Checkbox.constants.ts

```diff
@@ -0,0 +1,7 @@
+import { TCheckboxSizeToken } from './Checkbox.types';
```

Comment:
according to the previous comment we should add `CHECKBOX_ICON_SIZE_MAP`, `CHECKBOX_LABEL_SIZE_MAP` and `CHECKBOX_INPUT_SIZE_MAP` (for the variable `checkboxSize`)

Reply:
Why do we need the input size map? It's hidden

Reply:
`CHECKBOX_INPUT_SIZE_MAP` would be for the styled input